core/src/main/scala/codesearch/core/index/HaskellIndex.scala

```diff
@@ -91,7 +68,7 @@ class HaskellIndex(val ec: ExecutionContext) extends LanguageIndex[HackageTable]
     lastVersions
   }
 
-  private def contentByURI(uri: String): Option[(String, String, Result)] = {
+  override protected def contentByURI(uri: String): Option[ContentByURI] = {
```

Comment:
This isn't "ContentByURI" actually, it really just parses a csearch output line. If you're touching this code anyway, you can rename it while you're at it :slightly_smiling_face: Or leave a docstring. (And hey, even if you rename it, leave a docstring anyway.)